src/app/static/src/app/store/modelOutput/modelOutput.ts

```diff
@@ -14,19 +14,19 @@ export interface ModelOutputState {
 
 export const modelOutputGetters = {
     barchartIndicators: (state: ModelOutputState, getters: any, rootState: RootState): BarchartIndicator[] => {
-        return rootState.modelRun.result!.plottingMetadata.barchart.indicators;
+        return rootState.modelCalibrate.result!.plottingMetadata.barchart.indicators;
     },
     barchartFilters: (state: ModelOutputState, getters: any, rootState: RootState): Filter[] => {
        return outputPlotFilters(rootState);
     },
-    bubblePlotIndicators: (state: ModelOutputState, getters: any, rootState: RootState, rootGetters: any): ChoroplethIndicatorMetadata[] => {
-        return rootGetters['metadata/outputIndicatorsMetadata'];
+    bubblePlotIndicators: (state: ModelOutputState, getters: any, rootState: RootState): ChoroplethIndicatorMetadata[] => {
+        return rootState.modelCalibrate.result!.plottingMetadata.choropleth.indicators;
     },
-    bubblePlotFilters: (state: ModelOutputState, getters: any, rootState: RootState, rootGetters: any): Filter[] => {
+    bubblePlotFilters: (state: ModelOutputState, getters: any, rootState: RootState): Filter[] => {
         return outputPlotFilters(rootState);
     },
-    choroplethIndicators: (state: ModelOutputState, getters: any, rootState: RootState, rootGetters: any): ChoroplethIndicatorMetadata[] => {
-        return rootGetters['metadata/outputIndicatorsMetadata'];
+    choroplethIndicators: (state: ModelOutputState, getters: any, rootState: RootState): ChoroplethIndicatorMetadata[] => {
+        return rootState.modelCalibrate.result!.plottingMetadata.choropleth.indicators;
```

Comment:
Just wondering about these changes - is it right that we no longer use the plotting metadata at all  but just use the indicators that come back on the result object? I know that @EmmaLRussell recently made changes to the equivalent getters for the inputs choropleth which use both the plotting metdata and the indiviual endpoints' metadata: https://github.com/mrc-ide/hint/pull/617 

@EmmaLRussell are you in a position to confirm the correct behaviour here?

Reply:
We do still need the indicators from the plotting metadata endpoint - it contains the full metadata we require to use the indicator (e.g. what its data columns, what value format to use) - what's returned by the data endpoint really just indicates which filters should be included.

Reply:
Actually looks like the full metadata is returned with the calibration result, so this change is correct. But this means that the `outputIndicatorsMetadata` getter in the `metadata` module is not longer used anywhere and can be removed.